code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -591,7 +591,7 @@
 	basic_modules = list(
 		/obj/item/assembly/flash/cyborg,
 		/obj/item/reagent_containers/borghypo/syndicate,
-		/obj/item/twohanded/shockpaddles/syndicate,
+		/obj/item/twohanded/shockpaddles/syndicate/cyborg,

```

Comment:
document this.

Reply:
it's literally the same thing as before, i just had to make it a subtype so the normal syndie paddles couldn't be dropped